python/ray/cluster_utils.py

```diff
@@ -316,3 +317,23 @@ def shutdown(self):
             self.remove_node(self.head_node)
         # need to reset internal kv since gcs is down
         ray.experimental.internal_kv._internal_kv_reset()
+
+
+def get_job_submission_client_cluster_info(
+        address: str, create_cluster_if_needed: bool) -> ClusterInfo:
+    """Get address, cookies, and metadata used for
+        ray.dashboard.modules.job.sdk.JobSubmissionClient.
+
+        Args:
+            address (str): Address without the module prefix that is passed
+                to ray.dashboard.modules.job.sdk.JobSubmissionClient.
```

Comment:
```suggestion
                to JobSubmissionClient.
```
In case it gets moved :)